src/templates/project-details.js

```diff
@@ -30,31 +35,61 @@ const DetailsPage = ({ pageContext: { project } }) => {
           </aside>
         </article>
 
-        <article className={s.overview} className={s.content_container}>
-          <div className={s.overview_content} className={s.main}>
+        <article className={classnames(s.overview, s.content_container)}>
+          <div className={classnames(s.overview_content, s.main)}>
             <h3 className={s.title_props}>Project Overview</h3>
             <div className={s.project_description}>{project.description}</div>
           </div>
-          <div className={s.data} className={s.side}>
+          <div className={classnames(s.data, s.side)}>
             <div className={s.stack}>
               <h2 className={s.sub_title}>Technology</h2>
               {project.stack.map((stacks, index) => {
                 return <span key={`teck_stacks_${index}`} className={s.technology_content}> {(index ? ', ' : '') + stacks} </span>
               })
-            }
+              }
             </div>
             <div className={s.tags_section}>
               <h2 className={s.sub_title}>Tags</h2>
-              {project.tags.map(tag => (
-                <div className={s.tags_parent}>
-                  <div className={s.tags_child}>
-                    {tag}
-                  </div>
+              {project.tags.map((tag, index) => (
+                <div className={s.tags_parent} key={`tag_index${index}`}>
+                  <div className={s.tags_child}>{tag}</div>
                 </div>
               ))}
             </div>
           </div>
         </article>
+
+        <article className={s.other_content_container}>
```

Comment:
We talked about moving blocks of HTML into Functional Components (stateless) to make them easier to read and maintain and to avoid large blocks of code in render function

Reply:
Done